packages/framework/pr_tools/PullRequestLinuxDriver.sh

```diff
@@ -47,6 +48,14 @@ function bootstrap_modules() {
         module load python/3.7.3
         get_python_packages pip3
 
+        module list
+    elif [[ ${on_rhel8} == "1" ]]; then
+        source /projects/sems/modulefiles/utils/sems-modules-init.sh
```

Comment:
I don't think this is needed.  It is likely run elsewhere.  If it were needed here, I think it would be needed at line 44 as well.

I run this manually before running genconfig.  The automated process handles this for you though.

Reply:
We do need the `source ...` here.  Weaver and vortex do not need this because `MODULEPATH` is set automatically on those systems such that the modules we need are available.  Some of the non-pipline jenkins jobs run this source in their build step

Reply:
Oh I missed it at line 61 (I thought I was looking at RHEL7 but was looking at Weaver).  Yes, we need it, I was wrong.